creation/web_base/condor_startup.sh

```diff
@@ -71,6 +71,10 @@ add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$config_file" | cu
 . "$add_config_line_source"
 error_gen=$(gconfig_get ERROR_GEN_PATH "$config_file")
 
+# Read the knobs coming from the frontend configuration for blackhole detection (GLIDEIN_BLACKHOLE_NUMJOBS and GLIDEIN_BLACKHOLE_RATE)
+glidein_blackhole_numjobs="$(gconfig_get GLIDEIN_BLACKHOLE_NUMJOBS "$config_file")"
+glidein_blackhole_rate="$(gconfig_get GLIDEIN_BLACKHOLE_RATE "$config_file")"
```

Comment:
Minor. Quotes are not needed `var=$()` is preserving spaces

Reply:
updated